Makefile

```diff
@@ -151,7 +151,8 @@ docker: build
 format:
 	./hack/format.sh
 
-test: build
+.PHONY: test
+test:
 	./hack/run-ut.sh
```

Comment:
how can we pass param to this script

Reply:
You can now pass param by specifying UT_TYPE.

```sh
make test UT_TYPE=--docker
make test UT_TYPE=--local
```